android/pom.xml

```diff
@@ -462,5 +476,18 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>javac-for-jvm18plus</id>
+      <activation>
+        <!--
+            In order to build and run the tests against JDK 19+, we need to pass java.security.manager=allow, to make
+            the deprecated 'java.lang.SecurityManager' available for use.
+         -->
+        <jdk>[18,]</jdk>
+      </activation>
+      <properties>
+        <test.add.args>-Djava.security.manager=allow</test.add.args>
+      </properties>
+    </profile>
```

Comment:
On JDK18+, the [SecurityManager](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/SecurityManager.html) needs extra urging to work with the testsuite, as described in google/guava#6425.